### PR TITLE
Use hyphens in long CLI flags to be consistent with GNU utils

### PR DESCRIPTION
**Pull request by tranzystorek-io**
Highlights:

- Switches word separator for flags from underscores `_` to hyphens `-`
- Uses `.visible_alias(...)` so that clap documents aliases automatically

For formatting, a github action can be set up that doesn't allow merging unformatted code.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -90,10 +90,10 @@ fn main() {
             .last(true)
             .help("The command to run"))
         .arg(Arg::with_name("ttl")
-            .long("time_to_live")
-            .alias("ttl")
+            .long("time-to-live")
+            .visible_alias("ttl")
             .default_value("60s")
-            .help("Duration the cached result will be valid. Alias: --ttl"))
+            .help("Duration the cached result will be valid for"))
```
Comment (dimo414) [inline src/main.rs]: I don't think "for" is helpful here, just leave the text as-is (without the "Alias").

**Comment by dimo414** [comment]
Ah, I didn't realize the bulk of this PR was just `cargo fmt`, and I had been delaying reviewing it until I could go through your changes in detail.

I appreciate the motivation for using a formatter, but I'd intentionally not run `cargo fmt` on the project so far. I might decide to set up formatting in the future but at a minimum I'd want to add enforcement and probably also tune the formatter via a rustfmt.toml as well.

The changes in 697f2092 LG though, so if you want to remove the `cargo fmt` commit I can merge that.

**Comment by tranzystorek-io** [comment]
@dimo414 agreed, I just reset the formatting commit :)

File: src/main.rs (hunk 1 above)
Comment (tranzystorek-io) [reply src/main.rs]: Done

**Pull request merged**